lib/flutter_calendar_carousel.dart

```diff
@@ -56,6 +56,7 @@ class CalendarCarousel<T extends EventInterface> extends StatefulWidget {
   final TextStyle? prevDaysTextStyle;
   final TextStyle? daysTextStyle;
   final TextStyle? nextDaysTextStyle;
+  final Color backGroundColor;
```

Comment:
`background` is a single word: https://dictionary.cambridge.org/dictionary/english/background
`G` should not be capitalized

Reply:
@massimilianochiodi Agree with the comment here!